.github/workflows/pr-title-linter.yml

```diff
@@ -0,0 +1,16 @@
+name: "Conventional Commit Validation PR Title"
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Should this be `master` as we don't have a `main` branch on the extension?

Reply:
Updated